core/domain/user_services.py

```diff
@@ -406,7 +406,10 @@ def record_user_started_state_editor_tutorial(user_id):
     _save_user_settings(user_settings)
 
 
-def update_email_preferences(user_id, can_receive_email_updates):
+def update_email_preferences(
```

Comment:
Yes thats right. This function is called only in those cases. Now we can make third arg explicit. We will pass defaulr feconf arg in signup. But what about preferences handler when it calls this function. I can think of two way :

1. We nake 3rd arg explicit and pass default feconf as arg wherever function is called. We will need a placeholder 3rd arg in preference handler which will be changed with the value received from frontend in 2nd milestone.

2. We keep it default and change 3rd arg in 2nd milestone because 2nd milestone will change this function calls. In former case we will have to change function args right now in this commit so that oppia stays in stable state

So which way do you prefer ?